source/extensions/filters/http/ext_proc/processor_state.h

```diff
@@ -27,8 +27,6 @@ class QueuedChunk {
 public:
   // True if this represents the last chunk in the stream
   bool end_stream = false;
-  // True if the chunk was actually sent to the gRPC stream
-  bool delivered = false;
```

Comment:
No need to maintain this delivered flag if we do not support mode changing after streaming data to ext_proc server already started.

So basically,  there are two data modes which may put data into the chunk_queue:
1) in STREAMED mode, every chunk in the queue is delivered.
2) in BUFFERED_PARTIAL mode, the first chunk in the queue is delivered. And the rest of the chunks is not delivered, and no need to be delivered.

BTW, delivered means the data is sent to the ext_proc server.